packages/access-api/src/routes/validate-email.js

```diff
@@ -134,6 +138,46 @@ async function session(req, env) {
     req.query.ucan,
     delegation.capabilities[0].nb.key
   )
+  if (req.method.toLowerCase() === 'post') {
+    const accessSessionResult = await validator.access(delegation, {
+      capability: Access.session,
+      principal: Verifier,
+      authority: env.signer,
+    })
+    if (accessSessionResult.error) {
+      throw new Error(
+        `unable to validate access session: ${accessSessionResult.error}`
+      )
+    }
+    const account = { did: () => accessSessionResult.audience.did() }
+    const agentPubkey = accessSessionResult.capability.nb.key
+    const wrappedKeyCanAsignForAccount = await ucanto.delegate({
+      issuer: env.signer,
+      audience: { did: () => agentPubkey },
+      capabilities: [
+        {
+          with: env.signer.did(),
+          can: 'access-api/delegation',
```

Comment:
```suggestion
          can: 'access/delegation'
```

Nit: It just occurred to me that you can use `access/delegate` here as spec-ed except with `audience` being  an `agent` as opposed to service.

Reply:
I'm down to change to that post-merge but otherwise I'd prefer to make this a meaningless `can`